org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterColorColor.java

```diff
@@ -452,15 +457,21 @@ private void updateBrightness(PercentType brightness) {
         HSBType oldHSB = lastHSB;
         HSBType newHSB = new HSBType(oldHSB.getHue(), oldHSB.getSaturation(), brightness);
         lastHSB = newHSB;
-        updateChannelState(newHSB);
+
```

Comment:
Probably you should also update the HSB when the device is off.

Instead of this..
```
HSBType newHSB = new HSBType(oldHSB.getHue(), oldHSB.getSaturation(), brightness);
```
Try this..
```
HSBType newHSB = new HSBType(oldHSB.getHue(), oldHSB.getSaturation(), currentOnOffState.get() ? brightness :0);
```